### DISCUSSION TITLE
Feature request: Setting Neutral Point in BarGauge

### DISCUSSION THREAD

**Discussion by marimelon** [grafana/grafana#60613] [Dashboards, Panels & Visualizations]
In Grafana 9.3.0([PR](https://github.com/grafana/grafana/pull/53989)) it is now possible to set Neutral Point on the Gauge dashboard.

I would like to do the same on BarGauge (and in the Table).

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
